http/rest-client-reactive/src/test/java/io/quarkus/ts/http/restclient/reactive/OpenshiftTLSRegistryIT.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.ts.http.restclient.reactive;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Tag;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+
+@Tag("QUARKUS-5664")
+@QuarkusScenario
+@Disabled("https://github.com/quarkus-qe/quarkus-test-framework/issues/1052")
+public class OpenshiftTLSRegistryIT extends TLSRegistryIT {
```

Comment:
JVM baremetal run sports `2025-05-09T07:27:13.1079813Z [WARNING] Tests run: 6, Failures: 0, Errors: 0, Skipped: 6, Time elapsed: 0 s -- in io.quarkus.ts.http.restclient.reactive.OpenshiftTLSRegistryIT` because there exclusion pattern is not applied (`Openshift` -> `OpenShift`)